src/annotator/guest.js

```diff
@@ -122,16 +122,13 @@ export default class Guest {
    * @param {HTMLElement} element -
    *   The root element in which the `Guest` instance should be able to anchor
    *   or create annotations. In an ordinary web page this typically `document.body`.
-   * @param {EventBus} eventBus -
```

Comment:
There is a reference to `EventBus` in the typedefs in this file which needs to be removed.

Reply:
Thanks, I remove it the reference.